docs/Analog.md

```diff
@@ -1,32 +1,83 @@
 
-Discussion related to schematics for analog subsystem:
-- Related screenshots
-- Hierarchy
+A table of the pin mappings for each analog connector is shown below:
+
+| Pin number | Signal name |
+|------------|--------|
+| 1 | Analog P1 |
+| 2 | Analog N1 |
+| 3 | +15 V |
+| 4 | GND |
+| 5 | -15 V |
+| 6 | GND |
+| 7 | Analog P2 |
+| 8 | Analog N2 |
+
+Location of these 4 analog connectors in the AMDC (stacked as 2 x 2) is shown below:
+
+<img src="images/amdc-analog-input-highlighted.svg" />
+
+More detailed information on the analog connectors can be found in the [datasheet](https://media.digikey.com/pdf/Data%20Sheets/Amphenol%20PDFs/RJSAE_Brochure.pdf).
+
+### Difference amplifiers
+Analog input signals should be conditioned before transmitting to ADC. First stage of this conditioning is to decrease the voltage level. These are implemented using 4 [INA2143UA](http://www.ti.com/lit/ds/symlink/ina143.pdf) difference amplifier chips. Each chip has two op amps, totaling in 8 op amps each of which receives one differential analog input. These chips need a ±15 V supply which they receive from AMDC. Each op amp is configured as a non-inverting amplifier and the relationship between its input and output is as follows:
 
-## PCB Layout
 
-Discussion related to PCB layout for analog subsystem:
-- What area of board is it on?
-- Noise??
+_V_<sub>i</sub> = _V_<sub>REF</sub>  + 0.1 (_V_<sub>i</sub><sup>+</sup> - _V_<sub>i</sub><sup>-</sup>)
 
-## Max Voltage Limitations
+where i = 1, 2, 3 ..., 8. (_V_<sub>i</sub><sup>+</sup> - _V_<sub>i</sub><sup>-</sup>) is bipolar differential input coming from analog connectors and _V_<sub>i</sub> is output of the op amp which is fed to the low-pass filter (next subsection). _V_<sub>REF</sub> = 2.048 V is supplied from AMDC and 0.1 gain is set using laser trimmed resistors during IC fab, so each resistance value internal to the [INA2143UA](http://www.ti.com/lit/ds/symlink/ina143.pdf) is very accurate. This results in very high CMRR. Differential input to the op amp can be between ±20 V (AC/DC, each input between ±10 V), which means the output can be in the range of 0.048 V and 4.048 V.
+
+More detailed information on the operating conditions of the op amp can be found in the [datasheet](http://www.ti.com/lit/ds/symlink/ina143.pdf).
+ 
+### Low-Pass Filters (LPFs)
+After analog input voltage levels are decreased using op amps, the next stage is to remove high-frequency noises using low-pass filters. To implement this, 8 simple first-order RC filters are used each for 1 analog input. The cutoff frequency of the filter is selected based on the resistance and capacitance values:
+
+_f_<sub>c</sub> = 1 / (2 π _RC_)
+
+The main purpose of these LPFs, however, is for anti-aliasing. They are set to 50 kHz (resistor with _R_ = 100 Ω and capacitor with _C_ = 33 nF) since sampling is nominally done at 100 kHz for ADC, and control loop is nominally at 10 or 20 kHz.

```

Comment:
I'd like to see us acknowledge that the nominal build of the AMDC sets these to 50 kHz, but that custom builds can use different RC values to obtain a different filter bandwidth.